extensions/ql-vscode/src/query-history/store/variant-analysis-dto-mapper.ts

```diff
@@ -0,0 +1,253 @@
+import {
```

Comment:
Should this file be called `query-history-variant-analysis-dto-mapper.ts`, to fit with `query-history-variant-analysis-dto.ts`?

(Also applies to the domain mapper)

Reply:
Good shout.